koku/api/migrations/0062_add_infra_map_account_region.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.23 on 2024-02-19 14:13
+from django.db import migrations
+from django.db import models
```

Comment:
Can we separate the migration changes into a PR separate from the code changes?

Reply:
We can, though is it necessary? Since these are additive null-able fields :smile_cat:

Reply:
The migration should be separate from the code.

Reply:
https://github.com/project-koku/koku/pull/4932